src/inspect_ai/tool/_tools/_web_browser/_web_browser.py

```diff
@@ -369,18 +410,28 @@ async def web_browser_cmd(cmd: str, *args: str) -> str:
         )
     else:
         response = parse_web_browser_output(result.stdout)
-        if "web_at" in response:
-            web_at = (
-                str(response.get("web_at")) or "(no web accessiblity tree available)"
-            )
-            # Remove base64 data from images.
-            web_at_lines = web_at.split("\n")
-            web_at_lines = [
-                line.partition("data:image/png;base64")[0] for line in web_at_lines
-            ]
-            web_at = "\n".join(web_at_lines)
-            store().set(WEB_BROWSER_AT, web_at)
-            return web_at
+        if "web_html" or "web_at" in response:
+            if output_format == CrawlerOutputFormat.HTML:
+                html_content = (
+                    str(response.get("web_html")) or "(no HTML content available)"
+                )
+                store().set(WEB_BROWSER_HTML, html_content)
+                return html_content
+            elif output_format == CrawlerOutputFormat.AT:
+                web_at = (
+                    str(response.get("web_at"))
+                    or "(no web accessiblity tree available)"
+                )
+                # Remove base64 data from images.
+                web_at_lines = web_at.split("\n")
+                web_at_lines = [
+                    line.partition("data:image/png;base64")[0] for line in web_at_lines
+                ]
+                web_at = "\n".join(web_at_lines)
+                store().set(WEB_BROWSER_AT, web_at)
+                return web_at
+            else:
+                raise ValueError(f"Unknown output format: {output_format}")
```

Comment:
Note that enum up above implies support for DOM and Pixels but if the user tries to use these they'll get an error here (not saying we should support them here, just supporting the argument to narrow that enum).